### ISSUE TITLE
Labeling editor field for accessibility doesn't work

### ISSUE THREAD

**Issue by radavis** [tinymce/tinymce-react#177]
**What is the current behavior?**

Labeling the input by id doesn't work. A `<textarea>` tag is added to the DOM with the `aria-hidden` attribute set.

Example:

```
<label htmlFor="text-input-field">Text Input Field</label>
<Editor id="text-input-field ... />
```

[codesandbox](https://codesandbox.io/s/tinymcereact-forked-fbxd5?fontsize=14&hidenavigation=1&theme=dark)

**What is the expected behavior?**

Screen reader should pick up the label for the editor, and read the label.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE or `tinymce-react`?**

tinymce-react v3.5

**Comment by EkimChau** [comment]
Hi radavis,

This will need to be a feature request, currently when the editor content is focused, screen readers should start announcing "Rich Text Area. Press ALT-0 for help."

We could add a TinyMCE configuration that allows a developer to set their own custom label for each instance of the editor.  So that aria announcements are also internationalised, the developer would need to add translations strings for each language their editor supports, otherwise it will fallback to the english string declared in the init

This proposal will be slightly different from your suggestion of using the <label> tag to declare.  TinyMCE uses the title tag in the content body which provides an equivalent behaviour when used with screen readers.

Does this proposal work for you?

**Comment by radavis** [comment]
That sounds like a helpful addition, from my perspective.

**Comment by tiny-james** [comment]
Internal reference: TINY-1264

**Comment by tiny-james** [comment]
The related TinyMCE feature has been completed adding a new setting `iframe_aria_text`. Therefore I am closing this issue as completed.
